specifyweb/frontend/js_src/lib/components/FormParse/webOnlyViews.ts

```diff
@@ -84,6 +84,37 @@ export const webOnlyViews = f.store(() =>
       'edit',
       ['name']
     ),
+    [adminUser]: autoGenerateViewDefinition(
+      tables.SpecifyUser,
+      'form',
+      'edit',
+      ['name', 'password', 'agents']
+    ),
+    [institution]: autoGenerateViewDefinition(
+      tables.Institution,
+      'form',
+      'edit',
+      ['name', 'code', 'isAccessionsGlobal', 'isSingleGeographyTree', 'isSecurityOn', 'isServerBased']
+    ),
+    [division]: autoGenerateViewDefinition(
+      tables.Division,
+      'form',
+      'edit',
+      ['name', 'abbrev']
+    ),
+    [discipline]: autoGenerateViewDefinition(
+      tables.Discipline,
+      'form',
+      'edit',
+      // Required field: institution id
+      ['name', 'type']
+    ),
+    [collection]: autoGenerateViewDefinition(
+      tables.Collection,
+      'form',
+      'edit',
+      ['collectionName', 'code', 'catalogNumFormatName']
```

Comment:
`CatalogNumFormatName` should let the user choose form a default list of formats